start-page/beta/index.html

```diff
@@ -2,39 +2,169 @@
+                <path d="M11.5 1.5a10 10 0 0 0-10 10 10 10 0 0 0 10 10 10 10 0 0 0 10-10 10 10 0 0 0-10-10zm0 4.5a5.5 5.5 0 0 1 5.5 5.5 5.5 5.5 0 0 1-5.5 5.5A5.5 5.5 0 0 1 6 11.5 5.5 5.5 0 0 1 11.5 6Z" opacity=".2"/>
+                <path d="M11.5 1c-2.377 0-4.57.794-6.332 2.129a1.496 1.496 0 0 0-1.805.24 1.498 1.498 0 0 0-.24 1.807A10.446 10.446 0 0 0 1 11.5c0 2.376.793 4.57 2.127 6.33a1.495 1.495 0 0 0 .236 1.813c.494.493 1.24.57 1.815.234A10.445 10.445 0 0 0 11.5 22c2.373 0 4.563-.792 6.322-2.123.575.336 1.32.26 1.815-.234a1.495 1.495 0 0 0 .236-1.813A10.447 10.447 0 0 0 22 11.5c0-2.374-.791-4.564-2.123-6.324a1.498 1.498 0 0 0-2.045-2.047A10.446 10.446 0 0 0 11.5 1Zm0 1c2.086 0 4.012.67 5.578 1.807l-2.896 2.896a5.459 5.459 0 0 0-5.364 0L5.922 3.807A9.463 9.463 0 0 1 11.5 2ZM3.8 5.93l2.897 2.896A5.46 5.46 0 0 0 6 11.5c0 .974.257 1.887.703 2.682l-2.896 2.896A9.463 9.463 0 0 1 2 11.5c0-2.082.668-4.006 1.8-5.57Zm15.4 0A9.463 9.463 0 0 1 21 11.5c0 2.086-.67 4.012-1.807 5.578l-2.896-2.894A5.46 5.46 0 0 0 17 11.5a5.46 5.46 0 0 0-.697-2.674l2.896-2.896ZM11.5 7c.762 0 1.48.189 2.107.521a1.494 1.494 0 0 0 .372 1.506c.408.409.99.533 1.503.373.33.627.518 1.34.518 2.1 0 .764-.19 1.482-.523 2.111a1.49 1.49 0 0 0-1.498.375 1.496 1.496 0 0 0-.377 1.494A4.49 4.49 0 0 1 11.5 16a4.49 4.49 0 0 1-2.102-.52 1.499 1.499 0 0 0-.377-1.496 1.494 1.494 0 0 0-1.498-.373A4.492 4.492 0 0 1 7 11.5c0-.76.188-1.473.518-2.1.514.16 1.095.036 1.503-.373.41-.409.532-.99.372-1.506A4.492 4.492 0 0 1 11.5 7Zm-2.674 9.303A5.46 5.46 0 0 0 11.5 17a5.46 5.46 0 0 0 2.674-.697l2.896 2.894A9.461 9.461 0 0 1 11.5 21a9.461 9.461 0 0 1-5.57-1.803Z"/>
+              </svg>
+              Question? Problem?
+            </h2>
+            <p>{{ _('Post your general Thunderbird usage questions or problems on the support page, where many knowledgeable users can help you.') }}</p>
+            <a href="https://support.mozilla.org/questions/new/thunderbird" class="support">
+              {{ _('Support Page') }}
+              <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 12 12" height="12" width="12">
+                <path d="M4 1C2.347 1 1 2.347 1 4v4c0 1.653 1.347 3 3 3h4c1.653 0 3-1.347 3-3v-.5a.5.5 0 0 0-.5-.5.5.5 0 0 0-.5.5V8c0 1.117-.883 2-2 2H4c-1.117 0-2-.883-2-2V4c0-1.117.883-2 2-2h.5a.5.5 0 0 0 .5-.5.5.5 0 0 0-.5-.5Zm2.5 0a.5.5 0 0 0-.5.5.5.5 0 0 0 .5.5h2.793L6.146 5.146a.5.5 0 0 0 0 .708.5.5 0 0 0 .708 0L10 2.707V5.5a.5.5 0 0 0 .5.5.5.5 0 0 0 .5-.5v-4a.5.5 0 0 0-.094-.29.5.5 0 0 0-.002-.005.5.5 0 0 0-.004-.004.5.5 0 0 0-.046-.055.5.5 0 0 0-.034-.023.5.5 0 0 0-.062-.045.5.5 0 0 0-.065-.035.5.5 0 0 0-.082-.025.5.5 0 0 0-.072-.012A.5.5 0 0 0 10.5 1Z"/>
+              </svg>
+            </a>
+            <p>{{ _('Beta questions, how to use it or feedback about beta, may be posted in the beta forum, where you can also compare experiences with other beta users.') }}</p>
```

Comment:
Kind of a run-on here. I would rewrite this:
`In the Beta Forum, questions such as how to use beta or feedback about it may be posted. You can also compare experiences with other beta users.`